_examples/apache-zeppelin-example.adoc

```diff
@@ -0,0 +1,81 @@
+= Apache Zeppelin  Example
+:page-link: apache-zeppelin-example
+:page-weight: 100
+:page-labels: [Notebook, Zeppelin, Spark, R, Pyspark, Scala, Java]
+:page-layout: application
+:page-menu_template: menu_tutorial_application.html
+:page-description: This is an example of how to use Apache Zeppelin
+:page-project_links: ["https://github.com/rimolive/zeppelin-openshift", "https://github.com/rimolive/zeppelin-notebooks"]
+
+[[introduction]]
+== Introduction
+
+Apache Zeppelin is an alternative to project http://jupyter.org/[Jupyter] to provide 
+a notebook to run analysis over a data and plot graphs in order to show to end users
+an orverview about the data.
+
+This example shows how to use the Zeppelin openShift template created to run on top of
+OpenShift, as well as provide 
+
+[[architecture]]
+== Architecture
+
+The central piece is the Apache Zeppelin, used to create notebooks. From zeppelin we will
+connect to the Spark cluster and run some analysis.
```

Comment:
i like the description, but i think we should be using the openshift/kubernetes language here, for example "This example contains multiple Pods that will be deployed, a Pod for the Zeppelin notebook, and 2 or more for the Spark cluster Pods."